docker/batch-init/README.md

```diff
@@ -0,0 +1,124 @@
+# Batch Initialization Docker
+
+## Setup
+
+### Install AWS CLI
+
+The AWS CLI is required when you are running the containers below locally or on a non-EC2 instance that does not have appropriate AWS roles attached. For more information on how to install the AWS CLI see [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
+
+### Install Docker
+
+First install Docker. Here is a detailed guide on installing Docker on Ubuntu: [How to install and Use Docker on Ubuntu 18.04](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-18-04).
+
+### Setup Docker experimentals
+
+This is an optional step, but is highly **recommended**. Docker has an experimental feature called squash. Squashing
+your Docker image during the build process will significantly reduce the size of your final Docker image. To leverage
+the `--squash` flag during the build process you must enable experimental Docker functions. 
+
+To enable, edit `/etc/docker/daemon.json` and insert the following:
+
+```json
+{
+	"experimental": true
+}
+```
+
+Save, restart Docker via `service docker restart` and confirm experimental is enabled on the server. The following command will return `true` if experimental is enabled, `false` otherwise. 
+```
+$ docker version -f '{{.Server.Experimental}}'
+```
+
+### Deploy Batch Single to AWS Batch
+
+The Batch Initializer assumes that the Batch Single Docker container has been deployed to ECR and the AWS Batch infrastructure has been appropriate deployed via CloudFormation. For more information on Batch Single, see the README.md in the `AIDA-Interchange-Format/docker/batch-single` directory. For more information on deploying the AWS Batch infrastructure with CloudFormation, see the README.md in `AIDA-Interchange-Format/infrastructure` directory. 
+
+## Batch Intializer
+
+The Batch Initializer Docker image is responsible for taking in a single AIF Validation archive submission, running validation on the submission contents and format, and submitting jobs to AWS Batch for validation. Also included in this repository is the Batch Scheduler Docker image. Batch Scheduler Docker is an extension of the Batch Initializter and is used for running Batch initializations on more than one AIF Validation archive submission that is stored on S3. See [Batch Scheduler](#Batch-Scheduler). 
```

Comment:
Typo: Batch Initializter should be Batch Initializer.

Reply:
fixed